src/ontology/ro-edit.owl

```diff
@@ -3953,6 +3956,7 @@ InverseObjectProperties(obo:RO_0002439 obo:RO_0002458)
 
 # Object Property: obo:RO_0002440 (symbiotically interacts with)
 
+AnnotationAssertion(obo:IAO_0000112 obo:RO_0002440 "whales and the barnacles that attach to them")
```

Comment:
@anitacaron suggests to use the "new" object label (e.g., symbiotically interacts with) 

> whales and the barnacles that attach to them

> whales and the barnacles that attach to them, meaning that whales symbiotically interact with barnacles.

Reply:
updated